Week1/homework/app.js

```diff
@@ -1,11 +1,133 @@
 'use strict';
 
 {
+  // Task 1.1
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'macbeth',
+    'so_long_a_letter',
+    'silas_marner',
+    'julius_caesar',
+    'the_gods_are_not_to_blame',
+    'things_fall_apart',
+    'chike_and_the_river',
+    'a_man_for_all_seasons',
+    'installing_and_configuring_windows_10',
+    'installing_and_configuring_windows_server_2012',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // Task 1.4
+  const booksObject = {
+    macbeth: {
+      title: 'macbeth',
+      language: 'English',
+      author: 'William Shakespeare',
+    },
+    so_long_a_letter: {
+      title: 'so_long_a_letter',
+      language: 'French',
+      author: 'Mariama Ba',
+    },
+    silas_marner: {
+      title: 'silas_marner',
+      language: 'English',
+      author: 'George Eliot',
+    },
+    julius_caesar: {
+      title: 'julius_caesar',
+      language: 'English',
+      author: 'William Shakespeare',
+    },
+    the_gods_are_not_to_blame: {
+      title: 'the_gods_are_not_to_blame',
+      language: 'English',
+      author: 'Ola Rotimi',
+    },
+    things_fall_apart: {
+      title: 'things_fall_apart',
+      language: 'English',
+      author: 'Chinua Achebe',
+    },
+    chike_and_the_river: {
+      title: 'chike and_the_river',
+      language: 'English',
+      author: 'Chinua Achebe',
+    },
+    a_man_for_all_seasons: {
+      title: 'a_man_for_all_season',
+      language: 'English',
+      author: 'Robert Bolt',
+    },
+    installing_and_configuring_windows_10: {
+      title: 'installing_and_configuring_windows_7',
+      language: 'English',
+      author: 'Andrew Bettany & Andrew Warren',
+    },
+    installing_and_configuring_windows_server_2012: {
+      title: 'installing_and_configuring_windows_server_2012',
+      language: 'English',
+      author: 'Craig Zacker',
+    },
+  };
+
+  // Task 1.3 - 1.5
+  function actualBooksInfo() {
```

Comment:
For the naming of functions it's better to use a verb, to show it's an action. In this case you could rename it into "renderBookList" or "displayBookDetails".